specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -2928,6 +3008,27 @@
         "properties"
       ]
     },
+    "IntegrationRuntimeOperationStatus": {
```

Comment:
>IntegrationRuntimeOperationStatus [](http://example.com/codeflow?start=5&length=33)

You should use the standard type "OperationStatusResult" from common-types v4. See: [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/types.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/types.json). Don't define it locally.